src/components/CustomInput.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="custom-input">
+    <input 
+      id="answer" 
```

Comment:
Pass a custom prop here to give it a dynamic ID, so just something like `name` - This is for when you're selecting from a group of radios or checkboxes. So your component would then look like:

<CustomInput id="character" type="radio" text="Character 1" />
<CustomInput id="character" type="radio" text="Character 2" />
<CustomInput id="character" type="radio" text="Character 3" />

And also in order for this to work you'll need to add this to your input `:value="text"` this gives each input their own value. Let me know if that makes sense!